manifests/candlepin.pp

```diff
@@ -31,6 +28,8 @@
   }
 
   $java_client_cert_name = 'java-client'
+  $artemis_alias = 'artemis-client'
+  $artemis_client_dn = "C=${city}, ST=${state}, O=candlepin, OU=${org_unit}, CN=${hostname}"
```

Comment:
It actually seems like the order of these matters.... (most specific -> least specific?)

Should be like this: `CN=localhost, OU=SomeOrgUnit, O=candlepin, ST=North Carolina, C=US
`

Note also that C is country, not city

Reply:
Ahh yea, sorry you are right on the country.

I modeled this off of "C=US, ST=North Carolina, O=candlepin, OU=SomeOrgUnit, CN=dhcp-8-29-228.lab.eng.rdu2.redhat.com"

```
openssl x509 -noout -text -in /etc/pki/katello/certs/java-client.crt
```

Reply:
All good. When I re-order them in login.config it prevents Katello from connecting with an auth failure :shrug: